src/app/product/components/product-detail-page/product-details/product-details.component.ts

```diff
@@ -24,7 +24,7 @@ import { forEach } from '@angular/router/src/utils/collection';
 })
 export class ProductDetailsComponent implements OnInit, OnChanges {
   @Input() product: Product;
-  @Input() reviewList;
+
   dynamic = 50;
```

Comment:
what is this `dynamic = 50`? Static value?

Reply:
It is rating progress bar value at this state I have put it static. But we get these values from the database.

Reply:
done